dashboard/app/controllers/projects_controller.rb

```diff
@@ -133,22 +132,13 @@ def get_from_cache(key)
     @@project_level_cache[key] ||= Level.find_by_key(key)
   end
 
-  # Redirect to home if user not authenticated
-  def redirect_unless_account_type_requirements_met
-    admin_required = STANDALONE_PROJECTS[params[:key]][:admin_required]
-    user_is_admin = current_user.try(:admin?)
-    if admin_required && !user_is_admin
-      redirect_to '/'
-      return true
-    end
-
-    student_of_admin_required = STANDALONE_PROJECTS[params[:key]][:student_of_admin_required]
-    user_is_student_of_admin = current_user.try(:student_of_admin?)
-    if student_of_admin_required && !(user_is_admin || user_is_student_of_admin)
+  def redirect_unauthorized
```

Comment:
So I think CanCan should handle you not being signed in - at least it does for PLC. If you aren't signed in, it will make you sign in before giving you the resource or redirecting to not authorized.

If this doesn't work, let me know and we can debug this a little